Python install script doesn't compile on Galaxy

On Galaxy (one of the cluster of the Pawsey computer centre), there's no boost python. This is fine, but it seems that the python install scripts don't handle a missing boost python properly:
 
```
CMake Error at CMake/PythonInstall.cmake:120 (install):
  install FILES given unknown argument
  "/group/mwa/software/aoflagger/v2.12.1/galaxy/lib/python2.7/site-packages/lofar/dppp/".
Call Stack (most recent call first):
  DPPP/CMakeLists.txt:27 (python_install)


WARNING, Boost-Python not found; PythonDPPP will not be built.
-- Configuring incomplete, errors occurred!

```

Just removing the python install things from line DPPP/CMakeLists.txt:27 fixes this, but obviously doesn't install pythondppp anymore.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,6 @@ cmake_minimum_required(VERSION 3.9)
 
 project(DP3)
 
-# Casacore has a separate CMake file in this directory
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/CMake)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++11")
@@ -18,6 +17,8 @@ include_directories(${CASACORE_INCLUDE_DIR})
 # Add old casacore include directory because LOFAR beam library did
 # not use the casacore/ prefix in the includes until 3.2 (Sep '18)
 include_directories(${CASACORE_INCLUDE_DIR}/casacore)
+
+find_package(Threads REQUIRED)
 
 #Prevent accidentally finding old BoostConfig.cmake file from casapy
 set(Boost_NO_BOOST_CMAKE ON)
@@ -160,7 +161,8 @@ set(EXTRA_LIBRARIES ${EXTRA_LIBRARIES}
   ${CASACORE_LIBRARIES}
   ${Boost_DATE_TIME_LIBRARY} ${Boost_FILESYSTEM_LIBRARY} ${Boost_SYSTEM_LIBRARY} ${Boost_PYTHON_LIBRARY}
   ${EXTRA_LIBRARIES}
-  ${PYTHON_LIBRARIES})
+  ${PYTHON_LIBRARIES}
+  Threads::Threads)
   
 #
 # DDECal dependencies

--- a/PythonDPPP/CMakeLists.txt
+++ b/PythonDPPP/CMakeLists.txt
@@ -1,8 +1,5 @@
-# Try to find Boost-Python.
-# If found, build this package. Otherwise give a warning.
-FIND_PATH(BOOST_PYTHON_FOUND "boost/python.hpp")
-
-if(BOOST_PYTHON_FOUND)
+# If Boost-Python found, build this package. Otherwise give a warning.
+if(${Boost_PYTHON_FOUND})
 
   include(PythonInstall)
 

